tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -406,20 +406,13 @@ impl From<env::VarError> for FromEnvError {
 impl fmt::Display for FromEnvError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self.kind {
-            ErrorKind::Parse(ref p) => p.fmt(f),
-            ErrorKind::Env(ref e) => e.fmt(f),
```

Comment:
what's the motivation for this change? it doesn't seem like `<impl Display>::fmt` should warn?

Reply:
Blind copy & paste!